compose_testable_vignettes.R

```diff
@@ -46,9 +46,19 @@ rdt.cmd.head <- function(first, path)
     "include.configuration=TRUE,\n",
     "reload.state=", !first, ",\n",
     "block={\n\n",
+    "loadNamespace <- function(package, lib.loc = NULL,\n",
```

Comment:
I think new functions and stuff should be outside of the Rdt call -- no point introducing extra noise, especially in the case of running small  benchmarks like shootout.